script/version_checker/lib/main.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
What do you think about putting this in the flutter_plugin_tools?

Reply:
Ah, did not realize that it was a thing. Makes more sense for this to live there. Will add it.